### PR TITLE
Tooltip

### PR DESCRIPTION
**Pull request by ezywebs**
added tooltip with details

### CODE REVIEW

File: src/components/leaderBoard.js [hunk 1]
```diff
@@ -2,25 +2,49 @@ import React, { Component } from 'react';
...
 
+import { Popover, OverlayTrigger } from 'react-bootstrap';
+
 const CupArray = (props) => {
     return props.cups.map((cup) => {
         return <span>{cup}</span>
     })
 }
 
+function popoverHoverFocus(score){
```
Comment (charlesmwray) [inline src/components/leaderBoard.js]: please use function expression. see line 7

File: src/components/leaderBoard.js [hunk 2]
```diff
@@ -2,25 +2,49 @@ import React, { Component } from 'react';
...
+    return (
+        <Popover id="popover-trigger-hover-focus" title={score.username+" score details"}>
+                        Top score: <b>{ Math.max.apply(Math, score.scores) }</b><br/>
```
Comment (charlesmwray) [inline src/components/leaderBoard.js]: indentation is off here. please move back so that "Top" is indented once from <Popover

**Comment by charlesmwray** [comment]
This is a great update!

**Review by charlesmwray — CHANGES_REQUESTED** [review]

File: src/components/leaderBoard.js (hunk 1 above)
Comment (ezywebs) [reply src/components/leaderBoard.js]: change const to function? or wise versa?

**Comment by ezywebs** [comment]
did the changes, but I don't think I know how to send it for second review if there's such function at all lol

**Pull request merged**